src/pykeen/pipeline.py

```diff
@@ -659,6 +659,9 @@ def pipeline(  # noqa: C901
             experiment_id=mlflow_experiment_id,
             experiment_name=mlflow_experiment_name,
         )
+
+    # TODO add additional logic for wandb tracking
```

Comment:
Add additional keyword arguments to the pipeline as well as logic here to instantiate the WANDBResultTracker

Reply:
Let's merge #69 first and then I will take care of this for you

Reply:
Okay this is taken care of